front/package.json

```diff
@@ -10,6 +10,7 @@
     "@nivo/line": "^0.80.0",
     "@nivo/tooltip": "^0.80.0",
     "@openapi-contrib/openapi-schema-to-json-schema": "^5.1.0",
+    "@osrd-project/netzgrafik-frontend": "^0.0.0",
```

Comment:
It seems like this has pulled in an old version (the first we've ever pushed to NPM). I think we should pin the latest version until we have an official package: https://www.npmjs.com/package/@osrd-project/netzgrafik-frontend?activeTab=versions